src/pages/swap/useSwapUrlParams.ts

```diff
@@ -1,26 +1,32 @@
 import { Dispatch, SetStateAction, useEffect } from 'react';
 import { UseFormReturn } from 'react-hook-form';
 import { SwapFormValues } from '../../components/Nabla/schema';
+import { OutputTokenType } from '../../constants/tokenConfig';
+import { useVortexAccount } from '../../hooks/useVortexAccount';
 
 interface UseSwapUrlParamsProps {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   form: UseFormReturn<SwapFormValues, any, undefined>;
   setShowCompareFees: Dispatch<SetStateAction<boolean>>;
 }
+const defaultAmounts: Record<OutputTokenType, number> = { eurc: 1000, ars: 200 };
```

Comment:
Nit: let's maybe rename to 
```suggestion
const defaultFromAmounts: Record<OutputTokenType, number> = { eurc: 1000, ars: 200 };
```